sherpa-onnx/java-api/src/com/k2fsa/sherpaonnx/OnlineRecognizer.java

```diff
@@ -0,0 +1,290 @@
+/*
+ * // Copyright 2022-2023 by zhaoming
+ * // the online recognizer for sherpa-onnx, it can load config from a file
+ * // or by argument
+ */
+/*
+usage example:
+    String cfgpath=appdir+"/modelconfig.cfg";
+    OnlineRecognizer.setCfgPath(cfgpath);      //set model config file
+    OnlineRecognizer rcgOjb = new OnlineRecognizer();   //create a recognizer
+    CreateStream streamObj=rcgOjb.CreateStream();       //create a stream for read wav data
+    float[] buffer = rcgOjb.readWavFile(wavfilename); // read data from file
+    streamObj.acceptWaveform(buffer, 16000);          //feed stream with data, and sample rate is 16000
+    streamObj.inputFinished();                   //tell engine you done with all data
+    while (rcgOjb.IsReady(streamObj)) {          //engine is ready for unprocessed data
+
+                OnlineStream ssObj[]=new OnlineStream[1];
+                ssObj[0]=streamObj;
+                rcgOjb.DecodeStreams(ssObj);        //decode for multiple stream decode
+                //rcgOjb.DecodeStream(streamObj);   //decode for single stream decode
+            }
+
+    String recText = "simple:" + rcgOjb.GetResult(streamObj) + "\n";
+    byte[] utf8Data = recText.getBytes(StandardCharsets.UTF_8);
+    System.out.println(new String(utf8Data));
+    rcgOjb.Reset(streamObj);
+    rcgOjb.releaseStream(streamObj);       //release stream
+    rcgOjb.release();                      //release recognizer
+
+*/
+package com.k2fsa.sherpaonnx;
+
+import java.io.*;
+import java.util.*;
+
+public class OnlineRecognizer {
+    private long ptr = 0; // this is the asr engine ptrss
+
+    static private String cfgPath; // the config file, this file contains the model path and para , lib path

```

Comment:
Could you make it non-static? We can get its value from the constructor.

We can create different instances of OnlineRecognizer, each of which can use a different setting if we make this field non-static.